econet_hvac_furnace.yaml

```diff
@@ -23,6 +25,21 @@ econet:
             id(static_pressure).publish_state(((x[15] << 8) + x[16]) / 5280.0);
             id(return_air_temperature).publish_state(((x[50] << 8) + x[51]) / 10.0);
             id(flame_sensor).publish_state(x[33] / 10.0);
+    - sensor_datapoint: HRHEEMID
+      request_mod: 5
+      src_address: 448
+      datapoint_type: raw
+      then:
+        - lambda: |-
+            if(x.size() > 52) {
```

Comment:
Same as in the other PR. Space between if and parenthesis and negate to early return

Reply:
Updated.